docs/faq/frequently_asked_questions.rst

```diff
@@ -0,0 +1,56 @@
+
+#. Can voltage regeneration be disabled or reduced?
+    * Unfortunately, there is no way to completely prevent the module from regenerating current due to the limitations of physics. When attempting to step down the throttle with a load, such as a propeller, the module will always want to regenerate. However, how much current it regenerates depends on how aggressive the step-down is. There is a safety feature that is enabled by default that is designed to protect power supplies or other power sources that cannot absorb this current. When the supply voltage increases to a certain point, which generally happens when a power supply is being regenerated into, the module will begin to limit how aggressively the drive voltage changes to try and control how much current it is regenerating. While eliminating regeneration completely is impossible, Vertiq modules have options to limit the maximum amount of regeneration to protect power supplies. Please refer to our documentation about :ref:`Protecting Against Voltage Regeneration <protect_against_regen>` for more information.
+
+#. How do I get telemetry from my motors? How do I get Flight Controller logs?
+    * Please refer to our documentation about :ref:`Exporting Logs <ifci_log>`.
+    
+#. What causes electromagnetic interference (EMI) between the motor and the integrated ESC? Is electrical noise an issue when it comes to long communication wires? 
+    * Electromagnetic interference (EMI) can be radiated, coupled, or conducted at different frequencies while the motor is spinning. Electrical noise that is coupled to the motor’s body can be conducted to conduits carrying communications cables, such as the vehicle’s frame or drone arm. Longer communication wires can lead to more noise. To learn more about EMI and how to mitigate electrical noise, please refer to the :ref:`Understanding and Debugging Communication Errors from Motor Noise <motor_noise_debugging>` documentation.
+
+#. What flight controllers do you support? What firmware can I use (Ardupilot, PX4, etc.) ?
+    * Vertiq modules have been tested primarily with flight controllers using :ref:`PX4 <ifci_px4_flight_controller>` and :ref:`Ardupilot <dronecan_fc_tutorial>` firmware, and our documentation is focused on those types of flight controllers and their accompanying hardware. Other types of flight controllers may be used if they support protocols that can communicate with a Vertiq module, refer to your flight controllers documentation.
```

Comment:
point people to look at the Messaging Protocols section on the left side to see the list of things we support